docs/preview/03-Features/powershell/azure-active-directory.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Azure Active Directory"
+layout: default
+---
+
+# Azure Active Directory
+
+## Installation
+
+To have access to the following features, you have to import the module:
+
+```powershell
+PS> Install-Module -Name Arcus.Scripting.ActiveDirectory
+```
+
+## Access Rights to Azure Active Directory
+
+To interact with Azure Active Directory these scripts use the [Microsoft.Graph.Applications](https://learn.microsoft.com/en-us/powershell/module/microsoft.graph.applications/?view=graph-powershell-1.0) module, import this module:
+
+```powershell
+PS> Install-Module -Name Microsoft.Graph.Applications
+```
+
+After importing this module, make sure you are connected to Microsoft Graph with the following scopes:
+
+```powershell
+PS> Connect-MgGraph -Scopes "Application.Read.All,AppRoleAssignment.ReadWrite.All"
+```
+
+## Getting a list of Roles and Role Assignments for an Azure Active Directory Application
+
+Gets a list of roles and role assignments for an Azure Active Directory Application.
+
+| Parameter                 | Mandatory | Description                                                                                                                                                                                |
+| ------------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `ClientId`                | yes       | The client id of the Azure Active Directory Application Registration for which the roles and assignments are retrieved.                                                                    |
+| `RolesAssignedToClientId` | no        | The client id of the Azure Active Directory Application Registration to which roles have been assigned, used when you only want to retrieve the assignments for this specific application. |
+
+**Example**
+
+Retrieving all information for a Client Id.
+
+```powershell
+PS> Get-AzADAppRoleAssignments `
+-ClientId "b885c208-6067-44bd-aba9-4010c62b7d85" `
+#Found role 'FirstRole' on Active Directory Application 'main-application'
+#Role 'FirstRole' is assigned to the Active Directory Application 'client-application-one' with id '6ea09bbd-c21c-460c-b58a-f4a720f51826'
```

Comment:
Not sure what our policy is. Usually identity is written as capitals in user messages, I think. ID instead of id.

Reply:
Fixed in [`7f41876` (#336)](https://github.com/arcus-azure/arcus.scripting/pull/336/commits/7f418767ec7e50efda518c7af2ad94d8409fed4a)